### PR TITLE
add image classification example to optimum

### PR DESCRIPTION
**Pull request by prathikr**
# What does this PR do?

Adds image classification example to optimum

### CODE REVIEW

**Comment by prathikr** [comment]
@JingyaHuang can I please get a review on this PR? Thank you

File: examples/onnxruntime/training/image-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    --seed 1337
+```
+
+
+__Note__
+> *To enable ONNX Runtime training, your devices need to be equipped with GPU. Install the dependencies either with our prepared*
+*[Dockerfiles](https://github.com/huggingface/optimum/blob/main/examples/onnxruntime/training/docker/) or follow the instructions*
+*in [`torch_ort`](https://github.com/pytorch/ort/blob/main/torch_ort/docker/README.md).*
+
+> *The inference will use PyTorch by default, if you want to use ONNX Runtime backend instead, add the flag `--inference_with_ort`.*
```
Comment (JingyaHuang) [inline examples/onnxruntime/training/image-classification/README.md:53]: The support of ONNX export for VIT is not enabled yet. Can you remove `--inference_with_ort` note from README?

**Review by JingyaHuang — COMMENTED** [review]
Thanks for contributing the example @prathikr 🤗 ! Just left two small nits. Otherwise, it looks good to merge.

File: examples/onnxruntime/training/image-classification/README.md (hunk 1 above)
Comment (prathikr) [reply examples/onnxruntime/training/image-classification/README.md]: @JingyaHuang I tested ViT and I got ~5% gain from ORT with this command: python -m torch.distributed.launch --nproc_per_node=8 ~/optimum/examples/onnxruntime/training/image-classification/run_image_classification.py --model_name_or_path google/vit-base-patch16-224-in21k --dataset_name beans --overwrite_output_dir --output_dir ./outputs/ --remove_unused_columns False --do_train --do_eval --learning_rate 2e-5 --num_train_epochs 50 --per_device_train_batch_size 8  --per_device_eval_batch_size 8 --logging_strategy steps --logging_steps 10 --evaluation_strategy epoch --seed 1337 --fp16 True  --report_to none --optim adamw_ort_fused

Regardless though, I removed the --inference_with_ort flag since it isn't required for runs with optimum

Comment (JingyaHuang) [reply examples/onnxruntime/training/image-classification/README.md]: I suppose the gain is in training. Yeah `--inference_with_ort` is not implemented in the script so better remove it. Thanks.

**Comment by JingyaHuang** [comment]
Hi @prathikr, can you fix the code style so that I can merge it?

**Comment by prathikr** [comment]
@JingyaHuang done, thank you!

**Comment by JingyaHuang** [comment]
@prathikr Thanks for contributing!  The PR is merged.
